### PR TITLE
More validation and hints

### PR DESCRIPTION
**Pull request by EmilyGraceSeville7cf**
- check command substitution format
- suggest sample property names
- suggest `--`

### CODE REVIEW

File: schemas/completely.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
     "$schema": "http://json-schema.org/draft-07/schema",
-    "title": "completion",
-    "description": "A completion of the current application\nhttps://github.com/DannyBen/completely#using-the-completely-command-line",
-    "type": "object",
-    "patternProperties": {
-        ".": {
+    "definitions": {
+        "invalid-command-substitution": {
+            "type": "string",
+            "pattern": "\\$\\(\\s*\\)"
```
Comment (DannyBen) [inline schemas/completely.json:6]: What is this preventing? `$()`?
Is it important to prevent this specifically, and not other invalid strings?

Comment (EmilyGraceSeville7cf) [reply schemas/completely.json:6]: It prevents empty command substitutions like `echo $()`.

Comment (DannyBen) [reply schemas/completely.json:6]: Even `echo anything` is not expected to be there.
My point is, what is the point of limiting? This should be a simple list with all sorts of syntaxes available. Limiting can potentially prevent valid use cases that we haven't thought about.

I think that the original definition - list of strings - was sufficient to express what is expected.

Comment (EmilyGraceSeville7cf) [reply schemas/completely.json:6]: I thought an arbitrary shell command could be entered. I guess it's the root of my mistake. Should I close this PR?

**Pull request closed without merging**